tests.js

```diff
@@ -760,12 +768,13 @@ QUnit.test("Logical Operations", function(t) {
   this.assert_stream('or 0 (type "yup)', 'yup');
 });
 
-if (false) QUnit.test("Graphics", function(t) {
+QUnit.test("Graphics", function(t) {
   t.expect(69);
 
   // NOTE: test canvas is 300,300 (so -150...150 coordinates before hitting)
   // edge
 
+  this.interpreter.run('setspeed Infinity');
```

Comment:
Whoops, thanks. There is probably some other cruft remaining. But we should make sure that reports an error in the test harness. 

> On Feb 5, 2016, at 6:55 PM, Ian Bicking <notifications@github.com> wrote:
> 
> In tests.js:
> 
> >    t.expect(69);
> >  
> >    // NOTE: test canvas is 300,300 (so -150...150 coordinates before hitting)
> >    // edge
> >  
> > +  this.interpreter.run('setspeed Infinity');
> Is this unnecessary now that FD doesn't glide?
> 
> —
> Reply to this email directly or view it on GitHub.
>